spec/classes/qpid_spec.rb

```diff
@@ -29,6 +38,31 @@
           is_expected.to create_class('qpid').with_wcache_page_size(8)
         end
       end
+
+      context 'with enable_katello_agent false' do
+        let(:pre_condition) do
+          <<-PUPPET
+          class { 'katello::globals':
+            enable_katello_agent => false,
+          }
+          PUPPET
+        end
+
+        it do
+          is_expected.to create_class('qpid')
+            .with_ensure('absent')
+            .with_ssl(false)
+            .with_ssl_cert_db(nil)
+            .with_ssl_cert_password_file(nil)
+            .with_ssl_cert_name(nil)
+        end
+
+        it do
+          is_expected.not_to create_qpid__config__queue('katello.agent')
+            .with_ssl_cert('/etc/pki/katello/certs/foo.example.com-qpid-broker.crt')
+            .with_ssl_key('/etc/pki/katello/private/foo.example.com-qpid-broker.key')
```

Comment:
What is actually the behavior of

```
it do
  is_expected.not_to create_...('...')
    .with_...('...')
end
```
?

Intuitively it seems like adding `.with_...` wouldn't change much for the test. Is it sharper coverage to remove the `.with_...` and instead test that the cert/key pair itself doesn't get created?